content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- external-workspace-manager
+- gsoc
+author: alexsomai
+---
+
+Currently it's quite difficult to share and reuse the same workspace between multiple jobs and across nodes.
+There are some possible workarounds for achieving this, but each of them has its own drawback,
+e.g. stash/unstash pre-made artifacts, Copy Artifacts plugin or advanced job settings.
+A viable solution for this problem is the External Workspace Manager plugin, which facilitates workspace share and
+reuse across multiple Jenkins jobs and nodes.
+It also eliminates the need to copy, archive or move files.
+
+I'd like to announce that an alpha version of the External Manager Plugin has been released!
+It's now public available for testing.
+To be able to install this plugin, you must follow the steps from the Experimental Plugins Update Center
+link:/blog/2013/09/23/experimental-plugins-update-center/[blog post].
+The plugin's wiki page may be accessed
+link:https://wiki.jenkins-ci.org/display/JENKINS/External+Workspace+Manager+Plugin[here].
+
+The documentation that helps you get started with this plugin may be found on the
+link:https://github.com/jenkinsci/external-workspace-manager-plugin/blob/master/README.md[README] page.
+To get an idea of what this plugin does, which are the features implemented so far and to see a working demo of it,
+you can watch my mid-term presentation that is available link:https://youtu.be/u4zhxfUT8P4?t=22m7s[here].
+The slides for the presentation are shared on
+link:https://docs.google.com/presentation/d/1ZCYSIR2Tg466Ij1ghH5LSc8DLBCxWjIaD9IJcOyMZwU/edit?usp=sharing[Google Slides].
+
+I and my mentors, link:https://github.com/martinda[Martin] and link:https://github.com/oleg-nenashev[Oleg],
```

Comment:
> I and my mentors

should be:

> My mentors and I

Since there's links to Oleg and Martin, I would format it:

> My mentors, Martin and Oleg, and I have set up